app/reducers/article.js

```diff
@@ -12,26 +12,19 @@ function getInitialState() {
   return {
     article: {},
     currentArticle: null,
-    nextArticle: null,
-    topics: topicJsonMock.results
+    nextArticle: null
```

Comment:
I am not proposing to change #/article/$id at all.

I agree with you there will need to be an endpoint for getting the next work task. 
articleQueue is a more generic name than highlightQueue, which raises the question, does the more general name mean that it could return different types of tasks than Topic Highlighting? So could that endpoint return an article Quiz as well? Or is it just returning Topic Highlight tasks?

It is common to POST a write to a server, and then have the server make a decision about where to redirect the user next - (and from an API perspective, I think that would be the HATEOAS strategy)- I am wondering if we want to limit ourselves semantically to only giving the user another articleQueue task, or if we just say, the server can make an arbitrary decision about where to send the user next. 

For example, what do we do if the server runs out of TopicHighlight tasks? the server will want to send the user to a page that says "Nick's project is finished! Would you like to highlight topics for Alice's project or Bob's project next?"